test/e2e/cluster-logging/sd.go

```diff
@@ -26,24 +26,24 @@ import (
 	. "github.com/onsi/ginkgo"
 )
 
-var _ = framework.KubeDescribe("Cluster level logging using GCL", func() {
-	f := framework.NewDefaultFramework("gcl-logging")
+var _ = framework.KubeDescribe("Cluster level logging implemented by Stackdriver", func() {
```

Comment:
If I were just adding the tests, I would totally do this in a separate PR, but now I'm shuffling them around anyway, so I decided it's better to have it in one PR to avoid adding something I'll change moments later